server/chat.ts

```diff
@@ -1908,8 +1908,8 @@ export const Chat = new class {
 		}
 	}
 
-	getDataPokemonHTML(species: Species, gen = 7, tier = '') {
-		if (typeof species === 'string') species = Dex.deepClone(Dex.getSpecies(species));
+	getDataPokemonHTML(species: Species, gen = 8, tier = '') {
+		species = Dex.deepClone(species);
```

Comment:
Wait, why deepclone?

Reply:
Unsure, it was like that presumably before this file was even typescripted, given the species === string check

Reply:
I think that's probably safe to remove since this function doesn't do mutation.